microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -43,49 +43,49 @@ When you are interested in a particular DOM node, **Inspect** is a fast way to o
+   ![Highlighting the Michelangelo node](./index-images/michelangelo-elements-highlighted.png)
 
 1. Click the **Inspect** (![Inspect](./index-images/inspect-tool-icon-light-theme.png)) icon in the top-left corner of DevTools.
 
-   ![The Inspect icon](./index-images/dom-elements-highlighted-select-element-page-inspect.png)
+   ![The Inspect icon](./index-images/select-element-page-inspect.png)
 
 1. Under **Inspect a Node**, click the **Tokyo** text.  Now, `<p>Tokyo</p>` node is highlighted in the DOM Tree.
 
 Inspecting a node is also the first step towards viewing and changing the styles of a node.  See [Get started viewing and changing CSS](../css/index.md).
 
 
-### Navigate the DOM Tree with a keyboard
+#### Navigate the DOM Tree with a keyboard
 
 Once you have selected a node in the DOM Tree, you can navigate the DOM Tree with your keyboard.
 
 1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.
 
 1. In the rendered webpage, under **Navigate the DOM Tree with a Keyboard**, right-click **Ringo** and then select **Inspect**.  `<p>Ringo</p>` is selected in the DOM Tree.
 
-   ![Inspecting the 'Ringo' node](./index-images/dom-elements-highlighted-navigate-dom-tree-keyboard-ringo.png)
+   ![Inspecting the 'Ringo' node](./index-images/navigate-dom-tree-keyboard-ringo.png)
 
 1. Press the **Up Arrow** key two times.  `<div>` is selected.
 
-   ![Inspecting the 'div' node](./index-images/dom-elements-highlighted-navigate-dom-tree-keyboard-ul.png)
+   ![Inspecting the 'div' node](./index-images/navigate-dom-tree-keyboard-ul.png)
 
 1. Press the **Left Arrow** key.  The `<div>` list collapses.
 
 1. Press the **Left Arrow** key again.  The parent of the `<div>` node is selected.  In this case, it's the `<section>` that has the ID `navigate-the-dom-tree-with-a-keyboard-1`.
 
 1. Press the **Down Arrow** key two times, so that you have re-selected the `<div>` list that you just collapsed.
 
-   It should look like this: `<div>... </div>`
+   It should look like this: `<div>... </div>`.
 
 1. Press the **Right Arrow** key.  The list expands.
 
 
-### Scroll into view
+#### Scroll into view
 
-When viewing the DOM Tree, you may find yourself interested in a DOM node that isn't currently in the viewport.  For example, suppose that you scrolled to the bottom of the page, and you are interested in the `<h1>` node at the top of the page.  **Scroll into view** lets you quickly reposition the viewport so that you can review the node.
+When viewing the DOM Tree, you may find yourself interested in a DOM node that isn't currently visible in the rendered webpage.  For example, suppose that you scrolled to the bottom of the page, and you're interested in the `<h1>` node at the top of the page.  **Scroll into view** lets you quickly reposition the viewport so that you can review the node.
```

Comment:
```suggestion
When viewing the DOM Tree, you may find yourself interested in a DOM node that isn't currently visible in the rendered webpage.  For example, suppose that you scrolled to the bottom of the page, and you're interested in the `<h1>` node at the top of the page.  **Scroll into view** lets you quickly reposition the viewport so that you can see the node.
```
'review' has a particular narrow meaning that's probably not meant here